libraries/azqueue/test/db/versions/0001.yml

```diff
@@ -0,0 +1,153 @@
+version: 1
+migrationScript: |-
+  begin
+    create extension pgcrypto;
+    create table azure_queue_messages (
+      message_id uuid not null primary key,
+      queue_name text not null,
+      message_text text not null,
+      inserted timestamp not null,
+      visible timestamp not null, -- visible after this time
+      expires timestamp not null,  -- expired after this time
+      pop_receipt uuid -- null means not popped
+    );
+    -- 'get' operations sort by inserted within a queue
+    create index azure_queue_messages_inserted on azure_queue_messages(queue_name, inserted);
```

Comment:
In postgres, `select count(*)` has basically the same performance as `select *` (it loads each row to check its visibility in the current transaction).  The count operation is `select count(*) from azure_queue_messages msgs where msgs.queue_name = azure_queue_count.queue_name`, the where clause for which is a prefix of this index, so it will use the index to scan the rows for that queue name.

Reply:
```
2020-02-11 22:28:44.572 UTC [64] LOG:  duration: 0.204 ms  plan:
        Query Text: SELECT (select count(*)
          from azure_queue_messages msgs
          where msgs.queue_name = azure_queue_count.queue_name)
        Result  (cost=11.30..11.31 rows=1 width=8) (actual time=0.193..0.194 rows=1 loops=1)
          InitPlan 1 (returns $0)
            ->  Aggregate  (cost=11.29..11.30 rows=1 width=8) (actual time=0.188..0.190 rows=1 loops=1)
                  ->  Bitmap Heap Scan on azure_queue_messages msgs  (cost=4.17..11.28 rows=3 width=0) (actual time=0.020..0.107 rows=100 loops=1)
                        Recheck Cond: (queue_name = 'queue-1-DEF'::text)
                        Heap Blocks: exact=3
                        ->  Bitmap Index Scan on azure_queue_messages_inserted  (cost=0.00..4.17 rows=3 width=0) (actual time=0.013..0.013 rows=100 loops=1)
                              Index Cond: (queue_name = 'queue-1-DEF'::text)
```

That's an index scan looking for the queue name, and then a scan of the table based on the results of that index scan -- as expected.